src/renderers/shared/stack/reconciler/__tests__/ReactErrorBoundaries-test.js

```diff
@@ -84,12 +103,15 @@ describe('ReactErrorBoundaries', function() {
       }
     }
 
-    var EventPluginHub = require('EventPluginHub');
     var container = document.createElement('div');
-    EventPluginHub.putListener = jest.fn();
     container.innerHTML = ReactDOMServer.renderToString(<Boundary />);
     expect(container.firstChild.innerHTML).toBe('Happy Birthday!');
-    expect(EventPluginHub.putListener).not.toBeCalled();
+    // note: renderToString does not call componentDidMount
```

Comment:
Not sure why this even uses renderToString. Mind changing it to use ReactDOM.render so that we test the queueing behavior better? (See, I didn't even notice that the old test was testing the wrong thing but now that componentDidMount is completely absent here it's much more obvious. Thanks.)